core/navigation/src/main/kotlin/edu/stanford/spezi/core/navigation/DefaultNavigationEvent.kt

```diff
@@ -0,0 +1,6 @@
+package edu.stanford.spezi.core.navigation
+
+interface NavigationEvent
+sealed class DefaultNavigationEvent : NavigationEvent {
+    data object BluetoothScreen : DefaultNavigationEvent()
```

Comment:
correct. i would suggest leaving it as it is for now. the bluetooth view model currently has no repository that is injected, so the overhead would be comparatively large and i think the bluetooth screen will be adapted anyway, then we don't have to change it twice

Reply:
I am just wondering if this starts to introduce a coupling between the Bluetooth module and the core elements of what will be the Spezi framework. It could be convenient for now, but we will have to eventually make sure that the overall modules don't rely on implicit information or implementation details that can not be guaranteed in the future.

Reply:
This currently means that the BluetoothScreen NavigationEvent can theoretically be accessed even if the Bluetooth module is not present. This is currently unproblematic, as the BluetoothScreen is located in the application itself and therefore forces the BluetoothModule to be available. Even if the BluetoothScreen is later located in the Bluetooth module, this would not be a major problem. If there are reusable views for e.g. pairing devices, you would quickly realise that you cannot add a BluetoothScreen Composable in NavGraph if the module is missing.

My recommendation is to move the Bluetooth module from the core to the folder intended for specialised spezi modules. Not every application needs it, and in my opinion it does not belong in the core. but as I said, it is still not perfect at the moment, but will be changed as soon as the right views are ready for it :) @PSchmiedmayer